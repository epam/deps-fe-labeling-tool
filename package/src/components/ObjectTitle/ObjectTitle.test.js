import React from 'react'
import { shallow } from 'enzyme'
import { ObjectTitle } from '.'

const mockTitle = 'Coordinates'

describe('Component: ObjectTitle', () => {
  let defaultProps
  let wrapper

  beforeEach(() => {
    defaultProps = {
      title: mockTitle
    }

    wrapper = shallow(<ObjectTitle {...defaultProps} />)
  })

  it('should render correct layout', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
