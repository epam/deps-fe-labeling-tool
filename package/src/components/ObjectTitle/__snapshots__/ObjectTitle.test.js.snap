// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: ObjectTitle should render correct layout 1`] = `
<ObjectTitlestyles__Wrapper>
  Coordinates
</ObjectTitlestyles__Wrapper>
`;
